pkg/gadgettracermanager/gadgettracermanager.go

```diff
@@ -303,6 +303,41 @@ func (g *GadgetTracerManager) LookupMntnsByPod(namespace, pod string) map[string
 	return ret
 }
 
+// LookupPIDByContainer returns the PID of the container
+// specified in arguments or zero if not found
+func (g *GadgetTracerManager) LookupPIDByContainer(namespace, pod, container string) uint32 {
+	for _, c := range g.containers {
```

Comment:
gadgettracermanager.go declared a mutex to protect the hash maps from concurrent access, but it was not used.

FYI I wrote a patch to use the mutex, including in your new functions, see commit "gadgettracermanager: fix mutex usage" at https://github.com/kinvolk/inspektor-gadget/pull/225/commits

Reply:
Ok, let's put your changes on top of PR #224 and then I will rebase this PR.